src/core/l10n.js

```diff
@@ -44,6 +44,11 @@ const base = {
 
 const ko = {
   abstract: "요약",
+  participate: "참여",
+  references: "참조",
+  toc: "목차",
+  example: "예시",
+  informative: "정보",
```

Comment:
These two should go `src/core/example.js` and `src/core/informative.js`.

Reply:
I should probably finish up https://github.com/w3c/respec/issues/2084 first

Reply:
resolved